src/hyphen/hyphenator.py

```diff
@@ -54,22 +51,28 @@ def __init__(self, language='en_US', lmin=2, rmin=2, compound_lmin=2,
         single or compound words
         '''
         file_path = dictools.install(language, directory=directory)
-        self.__hyphenate__ = hnj.hyphenator_(file_path, lmin, rmin, compound_lmin, compound_rmin)
+        try:
+            self.__hyphenate__ = hnj.hyphenator_(file_path, 
+                lmin, rmin, 
+                compound_lmin, compound_rmin)
+        except Exception as E:
```

Comment:
Is there any way we can avoid catching a generic Exception here and be more specific? Maybe it should be a simple ValueError?